draft-ietf-quic-transport.md

```diff
@@ -633,7 +632,7 @@ on the two endpoints agreeing on a version.
 A QUIC connection begins with a client sending a handshake packet. Until the
 cryptographic handshake ({{handshake}}) produces 1-RTT keys, packets sent by a
 client MUST include the version in the packet header.  This allows the server to
-identify early packets and to enable version negotiation.
+identify 0-RTT packets and to enable version negotiation.
```

Comment:
I would simply remove the last sentence. I don't think that the server should be using the version bit to infer anything about what packets are being sent.